contrib/lang/clojure/packages.el

```diff
@@ -40,7 +40,8 @@ which require an initialization must be listed explicitly in the list.")
     :config
     (progn
       (when clojure-enable-fancify-symbols
-          (clojure/fancify-symbols))
+        (clojure/fancify-symbols 'clojure-mode)
+        (clojure/fancify-symbols 'cider-repl-mode))
```

Comment:
Better now?
What is the recommendend workflow for fixes to a PR?
I have pushed another commit that I intend to rewrite when you say it is ok.
Could I just have amended my previous commit and force-pushed it?